core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -593,7 +590,7 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
    * the given original sequence.
    *
    * @param origSequence the original sequence that is updated/created which generated the checks
-   * @param the components of the a sequence to check if they reference the original sequence
+   * @param components the components of the a sequence to check if they reference the original sequence
```

Comment:
Is this change intended?

Reply:
it's correcting a comment - want it in a separate pr?

Reply:
Nah, it's fine in this one. Just checking.